lib/plugins/aws/invokeLocal/index.js

```diff
@@ -200,7 +201,23 @@ class AwsInvokeLocal {
         );
       });
 
-      return BbPromise.all(promises).then(() => {
+      const fnRefPromises = _.entries(configuredEnvVars).map(([name, value]) => {
```

Comment:
It'll be nicer to use one`map` iteration for both `'Fn::ImportValue'` and `'Ref'`